test/e2e/data/e2e_eks_conf.yaml

```diff
@@ -216,6 +216,8 @@ variables:
   EXP_EKS_ADD_ROLES: "false"
   VPC_ADDON_VERSION: "v1.11.0-eksbuild.1"
   COREDNS_ADDON_VERSION: "v1.8.7-eksbuild.1"
+  KUBE_PROXY_ADDON_VERSION: "v1.8.7-eksbuild.1"
```

Comment:
@richardcase I shouldn't be changing this one but add a new config, right? 🤔 
And do I need a fully new suit like the one I added in eks_ipv6_test.go?